src/apis/core.ts

```diff
@@ -0,0 +1,54 @@
+import ky, { Input, Options } from "ky";
+
+import { getLocalStorage } from "@/utils/localStorage";
+
+const HTTP_METHODS = {
+  GET: "get",
+  POST: "post",
+  PATCH: "patch",
+  PUT: "put",
+  DELETE: "delete",
+} as const;
+
+type HttpMethod = "get" | "post" | "put" | "patch" | "delete";
+
+const kyInstance = ky.create({
+  prefixUrl: import.meta.env.VITE_BASE_URL,
+  timeout: 10000,
+  headers: {
+    "Content-Type": "application/json",
+  },
+  hooks: {
+    beforeRequest: [
+      (request) => {
+        const token = getLocalStorage("token");
+
+        if (token) {
+          request.headers.set("Authorization", `Bearer ${token}`);
+        }
+      },
+    ],
+    afterResponse: [
+      (_, __, response) => {
```

Comment:
```suggestion
      (_request, _options, response) => {
```

_를 1개, 2개 사용하여 구분하는것보다 어떤 파라미터인지 의미를 작성해 주는게 좋을 것 같습니다!

Reply:
아하 그런 방법이 있네요